Taarafo.Core/Startup.cs

```diff
@@ -25,38 +29,36 @@ public class Startup
 
         public void ConfigureServices(IServiceCollection services)
         {
+            LocalConfigurations localConfigurations = Configuration.Get<LocalConfigurations>();
+            services.AddScoped<LocalConfigurations>(sp => localConfigurations);
+
             services.AddLogging();
             services.AddControllers();
             services.AddDbContext<StorageBroker>();
             AddBrokers(services);
             AddServices(services);
-
-            services.AddSwaggerGen(options =>
-            {
-                var openApiInfo = new OpenApiInfo
-                {
-                    Title = "Taarafo.Core",
-                    Version = "v1"
-                };
-
-                options.SwaggerDoc(
-                    name: "v1",
-                    info: openApiInfo);
-            });
+            ConfigureOpenAPIDocument(services, localConfigurations);
         }
 
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        public void Configure(
+            IApplicationBuilder app,
+            IWebHostEnvironment env,
+            LocalConfigurations localConfigurations)
         {
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
                 app.UseSwagger();
+                IEnumerable<OpenAPIConfiguration> openAPIConfigurations = localConfigurations.OpenAPIs;
 
                 app.UseSwaggerUI(options =>
                 {
-                    options.SwaggerEndpoint(
-                        url: "/swagger/v1/swagger.json",
-                        name: "Taarafo.Core v1");
+                    foreach (var openAPIConfiguration in openAPIConfigurations)
+                    {
+                        options.SwaggerEndpoint(
+                            url: openAPIConfiguration.OpenAPIEndpoint.Url,
```

Comment:
We can remove the parameter aliasing as the ending property name is same as the parameter name.